### DISCUSSION TITLE
Constant Crashes

### DISCUSSION THREAD

**Discussion by psyouloveme** [WerWolv/ImHex#1656] [Get Help]
Hey, basically every time I try to use the projects feature the application starts crashing constantly. I'm trying to write really simple patterns to analyze stat data from a video game and I'm constantly losing progress. I have to save my project using the file menu (because the the keyboard shortcut does not work) **every** time I run my pattern script because flip a coin if imhex crashes instantly when I hit run. And the recovery feature completely deletes the pattern script half the time, so my process has to be edit the pattern script in a separate text editor and save it there, then copy and paste it into imhex, save the project, then hit the evaluate button and pray that it doesn't crash and corrupt the file I was looking at (which seems to happen idk every 5 crashes). That's not even as bad as trying to sort or resize columns - this crashes constantly as well. Half the time trying to copy a value of any kind it crashes. 

How is anyone getting any real work done with this program? It seems wildly unstable. I've tried multiple computers, different installations, different versions, I've tried portable, I've tried installing for all users, I've tried installing for only me. I've tried over and over and as much as I want to use this program it is not stable enough to use. 

Am I really the only person that's having constant, persistent crashes trying to do basic stuff with this program? What could I have possibly done wrong on 3 separate clean Windows installations that for at least 3 years I've found this program unfit for basic use because it's just so unstable. What am I missing? What could possibly be wrong with my systems? Is this program only designed for Linux and Mac?

**Comment by paxcut** [comment]
I just tested the latest master (1.33.0) on windows and Alt-S works for saving the project even if the pattern editor has focus.

**Comment by g-perist** [comment]
NO you are not the only one. It keeps hanging all the time. I wonder how well connected its creators are, It should have the worst reviews...

**Reply by WerWolv** [reply]
Terribly sorry that the free tool you downloaded from the internet for free was not up to your expectations. We'll issue a refund right away!
